Justfile

```diff
@@ -110,6 +110,16 @@ send-preconf:
 		--private-key 53321db7c1e331d93a11a41d16f004d7ff63972ec8ec7c25db329728ceeb1710 \
 		--slot head
 
+# manually send a preconfirmation to the bolt devnet
+send-preconf-count count:
+	cd bolt-spammer && RUST_LOG=info cargo run -- \
+		--provider-url $(kurtosis port print bolt-devnet el-1-geth-lighthouse rpc) \
+		--beacon-client-url $(kurtosis port print bolt-devnet cl-1-lighthouse-geth http) \
+		--bolt-sidecar-url http://$(kurtosis port print bolt-devnet mev-sidecar-api api)  \
+		--private-key 53321db7c1e331d93a11a41d16f004d7ff63972ec8ec7c25db329728ceeb1710 \
+		--slot head \
+		--count {{count}}
```

Comment:
can we have just one recipe with `count=1` as default? (same with blob command below)